packages/kbn-es/src/utils/docker.ts

```diff
@@ -309,16 +310,19 @@ export function resolveDockerImage({
  * Determine the port to bind the Serverless index node or Docker node to
  */
 export function resolvePort(options: ServerlessOptions | DockerOptions) {
+  const port = options.port || DEFAULT_PORT;
+  const value = ['-p', `127.0.0.1:${port}:${port}`];
+  const realLocalhostIp = getLocalhostRealIp();
+
+  if (realLocalhostIp) {
+    value.push('-p', `${realLocalhostIp}:${port}:${port}`);
```

Comment:
I'm curious why it was necessary to add real address while it wasn't necessary in https://github.com/elastic/kibana/pull/165415?

Reply:
because in previous case all the containers are running on the Docker and in this case we need to be able to reach the ES docker container from the VM running via Vagrant on the host machine